aws/resource_aws_msk_scram_secret.go

```diff
@@ -0,0 +1,216 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/kafka"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAwsMskScramSecret() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsMskScramSecretCreate,
+		Read:   resourceAwsMskScramSecretRead,
+		Update: resourceAwsMskScramSecretUpdate,
+		Delete: resourceAwsMskScramSecretDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"cluster_arn": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"secret_arn_list": {
+				Type:     schema.TypeList,
```

Comment:
QQ: is the order here important? wondering if we can use `TypeSet` here since we probably want a unique set of arns?

Reply:
if this can be a set (perhaps then change the name to `secret_arns`, it'll make for an easier object to filter out the necessary secrets with `Set` functions provided by the terraform plugin SDK (e.g. `Set.Intersection()`, `Set.Difference()`)